contracts/HypERC20Collateral.sol

```diff
@@ -28,14 +28,23 @@ contract HypERC20Collateral is TokenRouter {
         );
     }
 
-    function _transferFromSender(uint256 _amount) internal override {
-        require(wrappedToken.transferFrom(msg.sender, address(this), _amount));
-    }
-
-    function _transferTo(address _recipient, uint256 _amount)
+    function _transferFromSender(uint256 _amount)
         internal
         override
+        returns (bytes memory)
     {
-        require(wrappedToken.transfer(_recipient, _amount));
+        require(
+            wrappedToken.transferFrom(msg.sender, address(this), _amount),
+            "!transferFrom"
```

Comment:
Can we have a more expressive revert message here?

Reply:
such as? this is consistent with our other errors and there is no easy way to know the failure condition